pychromecast/controllers/dashcast.py

```diff
@@ -0,0 +1,62 @@
+"""
+Controller to interface with the DashCast app namespace.
+"""
+from ..config import APP_DASHCAST
+from . import BaseController
+
+
+APP_NAMESPACE = "urn:x-cast:com.madmod.dashcast"
+
+
+class DashCastController(BaseController):
+    """ Controller to interact with DashCast app namespace. """
+
+    # pylint: disable=useless-super-delegation
+    # The pylint rule useless-super-delegation doesn't realize
+    # we are setting default values here.
+    def __init__(self,
+                 appNamespace=APP_NAMESPACE,
```

Comment:
Why would you set default namespaces and not just hardcode them in the `super` call ?